src/plugins/mf_priority.cpp

```diff
@@ -198,12 +198,195 @@ int check_queue_factor (flux_plugin_t *p,
 }
 
 
+/*
+ * Add held job IDs to a JSON array to be added to a bank_info JSON object.
+ */
+static json_t *add_held_jobs (
+                            const std::pair<std::string, struct bank_info> &b)
+{
+    json_t *held_jobs = NULL;
+
+    // add any held jobs to a JSON array
+    if (!(held_jobs = json_array ()))
+        goto error;
+
+    for (auto const &j : b.second.held_jobs) {
+        json_t *jobid = json_integer (j);
+
+        if (!jobid)
+            goto error;
+
+        if (json_array_append_new (held_jobs, jobid) < 0) {
+            json_decref (jobid);
+            goto error;
+        }
+    }
+
+    return held_jobs;
+error:
+    json_decref (held_jobs);
+    return NULL;
+}
+
+
+/*
+ * Create a JSON object for a bank that a user belongs to.
+ */
+static json_t *pack_bank_info_object (
+                            const std::pair<std::string, struct bank_info> &b)
+{
+    json_t *bank_info, *held_jobs = NULL;
+
+    bank_info = json_object ();
+    if (!bank_info)
+        goto error;
```

Comment:
Oops I think this was left over from earlier?   The `bank_info` variable is overwritten by `json_pack()` further down, so the reference to the object created here is leaked.